packages/flutter/lib/src/semantics/semantics_service.dart

```diff
@@ -29,8 +29,12 @@ class SemanticsService {
   ///
   /// For example a camera application can use this method to make accessibility
   /// announcements regarding objects in the viewfinder.
-  static Future<void> announce(String message, TextDirection textDirection) async {
-    final AnnounceSemanticsEvent event = AnnounceSemanticsEvent(message, textDirection);
+  ///
+  /// The assertiveness level of the announcement can be determined by setting [assertivenessSetting].
+  /// Currently, this is only supported by the web engine and has no effect on other platforms.
```

Comment:
What do you get on other platforms? Always assertive? Always polite? Is it platform dependent?

Reply:
According to @chunhtai's comment above:

> Android does not have such setting, it will always interrupt. iOS has two different politeness, always gets interrupted or queue after current announcement.